tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/directives/common.directive.js

```diff
@@ -118,19 +118,27 @@
                                       <div class="FlyoutContent FlyoutHeading">\
                                           <div class="ms-Callout-content FlyoutHeadingText">  ' + obj.matterName + ' </div>\
                                        </div>\
-                                       <div class="ms-Callout-content commonFlyoutContaint">\
-                                          <div class="fontWeight600 ms-font-m FlyoutContentHeading">' + scope.$parent.$parent.$parent.grid.appScope.vm.matterConfigContent.GridColumn2Header + ':</div>\
+                                        <div class="ms-Callout-content commonFlyoutContaint"  ng-show=' + configs.search.searchColumnsUIPickerForMatter.matterClient.displayInFlyOut + '>\

```

Comment:
Displaying the information in the flyout, based on the property "displayInFlyOut"